src/python/grpcio/grpc/_cython/_cygrpc/channelz.pyx.pxi

```diff
@@ -0,0 +1,58 @@
+# Copyright 2018 The gRPC Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+cdef class Channelz:
+
+    def GetTopChannels(self, start_channel_id):
```

Comment:
Please name the functions `snake_case` like the rest of the code base. The only exception would be the actual `Servicer` implementation class, which adheres to different conventions due to the way the protobuf stub layer works. The functions in this file are internal and should be `snake_cased` instead.